react-native/debugging/handle-gradle-dependencies-clash.mo.md

```diff
@@ -0,0 +1,116 @@
+
+Then you must find out which version should be set.
+
+To do so, run `cd android && ./gradlew app:dependencies && cd ..`.
+This will print out your tree of dependencies.
+
+In there, you must look for two instances of the dependency but with different versions.
+For example if it returns:
+
+```
+_debugCompile - ## Internal use, do not manually configure ##
++--- project :react-native-maps
+|    +--- com.google.android.gms:play-services-base:11.1.6
+|    \--- com.google.android.gms:play-services-basement:11.1.6
++--- com.salesforce.marketingcloud:marketingcloudsdk:5.3.+ -> 5.3.1
+|    +--- com.google.android.gms:play-services-gcm:11.0.1 -> 11.1.6
+|    |    +--- com.google.android.gms:play-services-base:11.0.1 -> 11.1.6
+|    |    |    +--- com.google.android.gms:play-services-basement:11.0.1 -> 11.1.6
+|    |    |    |    \--- com.android.support:support-v4:25.2.0 -> 26.0.0 (*)
+|    |    |    \--- com.google.android.gms:play-services-tasks:11.0.1 -> 11.1.6
+|    |    |         \--- com.google.android.gms:play-services-basement:11.0.1 -> 11.1.6 (*)
+|    |    +--- com.google.android.gms:play-services-basement:11.0.1 -> 11.1.6 (*)
+|    |    \--- com.google.android.gms:play-services-iid:11.0.1 -> 11.1.6
+|    |         +--- com.google.android.gms:play-services-base:11.0.1 -> 11.1.6 (*)
+|    |         \--- com.google.android.gms:play-services-basement:11.0.1 -> 11.1.6 (*)
+|    +--- com.android.support:support-v4:26.0.0 (*)
+|    \--- com.android.support:support-annotations:26.0.0
+```
+
+A quick note on how to read this tree:
+
+- `(*)` indicates that this lib is already installed higher in the tree
+- `->` indicates that a different version of the library was installed
+
+You can see here for example that `com.google.android.gms:play-services-base` version is 11.1.6 for `react-native-maps` and version 11.0.1 for `com.salesforce.marketingcloud:marketingcloudsdk`.
+
+
+### 2. Force dependency to use a specific version
+
+So now in your `android/app/build.gradle` you need to:
+
+- Tell `react-native-maps` not to compile `com.google.android.gms:play-services-base`:
+```
+compile(project(':react-native-maps')){
+    exclude group: 'com.google.android.gms', module: 'play-services-base'
+}
+```
+
+- Tell `com.salesforce.marketingcloud:marketingcloudsdk` not to compile `com.google.android.gms:play-services-base`:
+```
+compile('com.salesforce.marketingcloud:marketingcloudsdk:5.3.+') {
+    exclude group: 'com.google.android.gms', module: 'play-services-base'
+}
+```
+
+- Force the version of `com.google.android.gms:play-services-base` to be the lowest one, i.e. 11.0.1:
+```
+compile ("com.google.android.gms:play-services-base:11.0.1") {
+  force = true;
+}
+```
+
+- **Run `cd android && ./gradlew clean && cd ..`** to update the dependencies
+
+
+> **Check:** your dependency tree should look like this:
+
+```
+_debugCompile - ## Internal use, do not manually configure ##
++--- project :react-native-maps
```

Comment:
Same thing as above, could you trim the tree a bit?

Reply:
yep